Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py

```diff
@@ -34,34 +34,59 @@ def create_relationships_from_entity(indicator_relationships):
     return entity_relationships
 
 
-def create_indicators_loop(indicators):
+def create_indicators_loop(args, indicators):
     """ Create indicators using createNewIndicator automation
 
     :param indicators: parsed indicators
     :return: errors if exist
     """
     relationships_objects = list()
     errors = list()
+    context_data = []
+    add_context = argToBoolean(args.get('add_context', False))
+    tags = argToList(args.get('tags'))
+
     for indicator in indicators:
         indicator['type'] = indicator.get('indicator_type')
         relationship_object = create_relationships_from_entity(indicator.get('relationships'))
         if relationship_object:
             relationships_objects.extend(relationship_object)
+
+        if len(tags) > 0:
+            if 'customFields' in indicator:
+                if 'tags' in indicator.get('customFields'):
+                    [indicator['customFields']['tags'].append(ntag) for ntag in tags]
+
         res = demisto.executeCommand("createNewIndicator", indicator)
+
+        context = {'value': indicator['value'], 'type': indicator['type']}
```

Comment:
consider adding the tags to the context too

Reply:
Done. Added tags to context output.